static/js/settings.js

```diff
@@ -318,7 +322,7 @@ function _setup_page() {
     $("#change_notification_settings").on("click", function (e) {
         var updated_settings = {};
         _.each(["enable_stream_desktop_notifications", "enable_stream_sounds",
-                "enable_desktop_notifications", "enable_sounds",
+                "enable_desktop_notifications","hide_private_message_desktop_notifications", "enable_sounds",
```

Comment:
the whitespace is off here; can you fix this to match the rest of the style?  Might just give it its own line (and same for `enable_sounds`).